custom-example/updateqrc.py

```diff
@@ -29,10 +29,16 @@ def process(src, exclusion, dst):
     file3.close()
 
 def main():
-    if(os.path.isfile(qgc_exc)):
-        process(os.path.join("../",qgc_rc), qgc_exc, qgc_rc)
-    if(os.path.isfile(res_exc)):
-        process(os.path.join("../",res_rc), res_exc, res_rc)
+    if (os.path.isfile(qgc_exc)):
+        qrc_path = os.path.join("../", qgc_rc)
+        if (!os.path.exists(qrc_path)):
```

Comment:
python not operator is 'not', '!' doesn't work. At least it doesn't on the python installed on ubuntu 18.04